assets/queries/terraform/aws/api_gateway_access_logging_disabled/query.rego

```diff
@@ -33,3 +171,17 @@ CxPolicy[result] {
 		"keyActualValue": "'access_log_settings' is not defined",
 	}
 }
+
+validNonEmptyKey(field, key) = output {
```

Comment:
Maybe we should add this function to the terraform rego lib. What do you think?

Reply:
Agreed